Tests/ArrayCoreTest.cpp

```diff
@@ -0,0 +1,62 @@
+#include "gtest/gtest.h"
+#include "plotly_maker/plotly_maker.h"
+#include "array_core/array_core.h"
+#include "common_utils/common_utils.h"
+#include <fstream>
+#include <windows.h>
```

Comment:
It is not a crossplatform include,
please use preprocessors directives
to handle different operation systems